sdk/keyvault/azure-keyvault-keys/tests/test_parse_id.py

```diff
@@ -0,0 +1,61 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# -------------------------------------
+import functools
+from azure.keyvault.keys import KeyClient, parse_key_vault_key_id
+from devtools_testutils import ResourceGroupPreparer, KeyVaultPreparer
+
+from _shared.preparer import KeyVaultClientPreparer as _KeyVaultClientPreparer
+from _shared.test_case import KeyVaultTestCase
+
+# pre-apply the client_cls positional argument so it needn't be explicitly passed below
+KeyVaultClientPreparer = functools.partial(_KeyVaultClientPreparer, KeyClient)
```

Comment:
Nitpicking, but this isn't really an improvement because this module only uses the preparer once.